.github/workflows/pull-fork.yml

```diff
@@ -0,0 +1,89 @@
+name: Pull request build
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+defaults:
+  run:
+    working-directory: src/client
+
+env:
+  BUCKET_WEB: reactive-trader-web-builds
+  BUCKET_OPENFIN: reactive-trader-openfin-builds
+  BUCKET_FINSEMBLE: reactive-trader-finsemble-builds
+
+jobs:
+  build:
+    name: Build & deploy
+
+    runs-on: ubuntu-20.04
+
+    if: "github.event.pull_request.header.repo.fork"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set Node to v16
```

Comment:
side note: after we have [tested locally](https://adaptive.kanbanize.com/ctrl_board/18/cards/4832/details/) and established that it is available on GH Actions, we should bump all of these to v20